example/app.js

```diff
@@ -5,7 +5,8 @@ var instantsearch = require('../');
 var search = instantsearch({
   appId: 'latency',
   apiKey: '6be0576ff61c053d5f9a3225e2a90f76',
-  indexName: 'instant_search'
+  indexName: 'instant_search',
+  numberLocale: 'fr-FR'
```

Comment:
I think the example is in english, but if this is for testing the feature why not